### PR TITLE
chore: fixed typo

### PR DESCRIPTION
**Pull request by hong4rc**
The test fails #8556

I only change the comment, doc and local variable name

### CODE REVIEW

File: cli/tsc/00_typescript.js [hunk 1]
```diff
@@ -7375,7 +7375,7 @@ var ts;
                     close: function () { return _fs.unwatchFile(fileName, fileChanged); }
                 };
                 function fileChanged(curr, prev) {
-                    // previous event kind check is to ensure we recongnize the file as previously also missing when it is restored or renamed twice (that is it disappears and reappears)
+                    // previous event kind check is to ensure we recognize the file as previously also missing when it is restored or renamed twice (that is it disappears and reappears)
```
Comment (kitsonk) [inline cli/tsc/00_typescript.js:7378]: Please do not modify this file, it is obtained upstream.  If you want to fix typos here, please consider contributing to microsoft/TypeScript and those will flow through to here.

File: op_crates/web/08_text_encoding.js [hunk 2]
```diff
@@ -1147,7 +1147,7 @@
     let outIndex = 0;
 
     let state = 0;
-    let codepoint = 0;
+    let codePoint = 0;
```
Comment (kitsonk) [inline op_crates/web/08_text_encoding.js:1150]: Please revert... "codepoint" is preferred.

**Review by kitsonk — CHANGES_REQUESTED** [review]

File: cli/tsc/00_typescript.js (hunk 1 above)
Comment (hong4rc) [reply cli/tsc/00_typescript.js]: Reverted this change

**Review by kitsonk — APPROVED** [review]
LGTM

**Pull request merged**